playbooks/learner-profile-dashboard.yml

```diff
@@ -19,6 +19,7 @@
       # Backup of logs is initiated by logrotate, which incidentally
       # also creates data to be backed up.
       TARSNAP_CRONTAB_STATE: "absent"
+      TARSNAP_BACKUP_GLOBAL_LOCK: false
```

Comment:
@tomaszgy Could you add the following comment above this line:

```
      # The backup lock is causing problems for LPD, since the lock is inherited by
      # gunicorn workers, which don't release it unless they are stopped.  Since the lock
      # is not needed for a daily backup anyway, we simply disable it.
```

The same info is already available in the [playbook](https://github.com/open-craft/ansible-playbooks/blob/35e9bdfe0fdb7e125806c356bd1d58a747118c49/playbooks/dalite.yml#L22-L24) for Dalite, but further down the line it probably won't be obvious where to look for it. So it would be worth repeating here :)

Reply:
Sure, done :)